cp1404practicals/prac_6/guitars.py

```diff
@@ -0,0 +1,32 @@
+from prac_6.guitar_class import Guitar
+
+
+def main():
+    print("My guitars!")
+    guitars = []
+    while True:
+        make = str(input("Make: >")).title()
+        if make == "":
+            break
+        model = str(input("Model: >")).title()
+        year = ""
+        while year == "":
+            try:
+                year = int(input("Year: >"))
+            except ValueError:
+                print("Year input must be an integer")
+        cost = ""
+        while cost == "":
+            try:
+                cost = float(input("Cost: >"))
+            except ValueError:
+                print("Cost input must be a number")
+        new_guitar_entry = Guitar(make, model, year, cost)
+        guitars.append(new_guitar_entry)
+
+    print("These are my guitars")
+    for i, guitar in enumerate(guitars, 1):
+        print(f"Guitar {i}: {guitar}")
```

Comment:
I think there is supposed to be some  specific formatting done here:
"Guitar 2: Gibson L-5 CES (1922), worth $ 16,035.40 (vintage)"
Things like the vintage function in the class is supposed to be used. 
Otherwise the while loops and try excepts are really good.